x-pack/plugins/security/server/audit/audit_events.ts

```diff
@@ -45,7 +45,7 @@ export interface AuditEvent {
      */
     saved_object?: {
       type: string;
-      id?: string;
+      id: string;
     };
```

Comment:
😁

Reply:
yes yes yes, I know what you're gonna say.. :D